dist/public.asset.php

```diff
@@ -1 +1 @@
-<?php return array('dependencies' => array(), 'version' => '24244cfb823cfde2c02a');
+<?php return array('dependencies' => array(), 'version' => 'c6c49a63c07f626533d5');
```

Comment:
just wondering if you are using these files, if not could be something we add to .gitignore. There intention is to determine what dependencies are required and auto load, but seems like the only thing they are doing is creating an extra file in every commit.